cmd/dao_tests/resource_test.go

```diff
@@ -0,0 +1,165 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func createPkResource() *models.PubkeyResource {
+	return &models.PubkeyResource{
+		Tag:      "tag1",
+		PubkeyID: 1,
+		Provider: models.ProviderTypeNoop,
+		Handle:   "handle1",
+	}
+}
+
+func createPkResource2() *models.PubkeyResource {
+	return &models.PubkeyResource{
+		Tag:      "tag1",
+		PubkeyID: 1,
+		Provider: models.ProviderTypeAzure,
+		Handle:   "handle1",
```

Comment:
I think handles might be unique in our database:

```suggestion
		Handle:   factories.GetSequenceName("handle"),
```

The same for other fixtures.